src/components/UI/ResourceWebinar/ResourceWebinar.tsx

```diff
@@ -41,31 +41,33 @@ export const ResourceWebinar: FunctionComponent<ResourceWebinarProps> = ({ title
             data-testid="youtubeEmbed-iframe"
           />
         </div>
-        <div className="w-full px-5 pt-3 pb-5">
+        <div className="w-full text-cloud-900 px-5 pt-3 pb-5">
           <h4 className="title mb-2">
             <a
-              className="text-gray-700 font-medium text-2xl"
               href={`https://www.youtube.com/watch?v=${youtubeEmbedCode}`}
               target="_blank"
               rel="noopener noreferrer"
+              className="text-cloud-900"
               data-testid="youtubeEmbed-title-link"
             >
               {title}
             </a>
           </h4>
-          <div className="inline-block border border-solid border-gray text-gray-500 w-auto text-sm rounded font-medium p-1 mb-4">
+          <div className="inline-block border border-solid border-gray w-auto text-sm rounded font-medium p-1 mb-4">
             {tag}
           </div>
           <p className="mb-4">{description}</p>
           <Dropdown
             data-testid="quick-video-links-dropdown"
             dropdownButtonText="Quick Video Links"
-            className="rounded border border-gray-300 text-gray-700 p-2 mb-2"
+            classNameShared="w-full"
+            className="rounded border border-gray-300 p-2 mb-2"
```

Comment:
```suggestion
            className="rounded border border-gray-300 p-2 mb-2"
```
remove `w-full`

Reply:
remove "w-full" will result in overflow for the mobile viewport. thus changing to w-full lg:w-auto instead